src/test/java/io/reactivex/flowable/FlowableTests.java

```diff
@@ -271,10 +271,7 @@ public Integer apply(Integer t1, Integer t2) {
         verify(w).onNext(10);
     }
 
-    /**
-     * A reduce should fail with an NoSuchElementException if done on an empty Observable.
-     */
-    @Test(expected = NoSuchElementException.class)
+    @Test
```

Comment:
Shouldn't the body of this test change too? Maybe to `test().assertComplete()` or whatever?